lib/fhir_models.rb

```diff
@@ -1,44 +1,36 @@
-# Top level include file that brings in all the necessary code
-require 'bundler/setup'
+require 'fhir_models/version'
+
+# TODO what are these for?  Nothing seems to use them.
 require 'yaml'
-require 'nokogiri'
 require 'date_time_precision'
 require 'date_time_precision/format/iso8601'
-require 'mime/types'
```

Comment:
mime/types and bcp47 are used for value set validation. Big decimal is used in hashable. logger is used for logging. tempfile is used in tests.